packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js

```diff
@@ -209,6 +209,29 @@ QUnit.module('Chat', () => {
 
             assert.strictEqual(messageList.option('showDayHeaders'), false, 'showDayHeaders is passed on runtime');
         });
+
+        QUnit.test('Chat should pass messageTemplate to messageList on init', function(assert) {
+            const messageTemplate = () => { return $('<div>'); };
+            this.reinit({
+                messageTemplate,
+            });
+
+            const messageList = this.getMessageList();
+            assert.strictEqual(messageList.option('messageTemplate'), messageTemplate, 'messageList is passed on init');
+        });
+
+        QUnit.test('Chat should pass messageTemplate to messageList at runtime', function(assert) {
+            this.reinit({ });
+
+            const messageTemplate = () => { return $('<div>'); };
+
+            this.instance.option('messageTemplate', messageTemplate);
+
+            const messageList = this.getMessageList();
+
+
+            assert.strictEqual(messageList.option('messageTemplate'), messageTemplate, 'messageTemplate is passed on runtime');
+        });
```

Comment:
[text coverage]
Let's not at least add a test checking that template data is expected. 
In a separate PR we will need to test template function call count, isLast functionality and correct passing to nested components

Reply:
added couple tests